Sources/AppcuesKit/AppcuesKit.docc/AppcuesKit.md

```diff
@@ -44,6 +44,7 @@ A Swift library for sending user properties and events to the Appcues API and re
 - <doc:Traits>
 
 ### Element Targeting
```

Comment:
Thinking this heading maybe should be "Anchored Tooltips"? (my bad)

Actually, maybe move this doc reference up under Core Functionality and leave this heading as the details for element targeting. This section is buried under Extending Experiences since it's the sort of stuff that almost nobody caries about.

Reply:
yep, i like that. updating